hive-writer/hive-writer.py

```diff
@@ -20,16 +21,22 @@
 # BOL: Switching off TestNet, we should test on Hive for now.
 USE_TEST_NODE = os.getenv("USE_TEST_NODE", 'False').lower() in ('true', '1', 't')
 TEST_NODE = ['http://testnet.openhive.network:8091']
-CURRENT_PODPING_VERSION = "0.2"
-NOTIFICATION_REASONS = ['feed_update','new_feed','host_change']
-HIVE_OPERPATION_PERIOD = 3  # 1 Hive operation per this period in
-MAX_URL_PER_CUSTOM_JSON = 130 # total json size must be below 8192 bytes
+CURRENT_PODPING_VERSION = 2
+NOTIFICATION_REASONS = {
+    'feed_update' : 1,
+    'new_feed' : 2,
+    'host_change' : 3
+}
+
+
+HIVE_OPERATION_PERIOD = 3       # 1 Hive operation per this period in
+MAX_URL_PER_CUSTOM_JSON = 130   # total json size must be below 8192 bytes
 
 # This is a global signal to shut down until RC's recover
 # Stores the RC cost of each operation to calculate an average
 HALT_THE_QUEUE = False
 # HALT_TIME = [1,2,3]
-HALT_TIME = [0,3,9,12,30,60,120]
+HALT_TIME = [0,1,1,3,6,9,15]
```

Comment:
What's the reasoning for reducing HALT_TIME so dramatically? 
If so, should additional HALT_TIME values be added?
HALT_TIME=[01,1,3,6,9,15,30,60,120]